panel/src/components/Forms/Blocks/Block.vue

```diff
@@ -202,6 +202,17 @@ export default {
 		}
 	},
 	methods: {
+		backspace(e) {
+			// ignore the shortcut when an input is focused
+			if (e.target.matches("[contenteditable], input, textarea")) {
+				return false;
+			}
+
+			e.preventDefault();
+			e.stopPropagation();
```

Comment:
Isn't this redundant since the line above still has `.stop`?

Reply:
You are right. It's redundant.